test-infrastructure/screenplay/interactions/handle.cookie.banner.js

```diff
@@ -7,7 +7,9 @@ export default class HandleCookieBanner extends Task {
   }
 
   async performAs(actor) {
-    const exemption = actor.recalls('exemption')
+    const exemption = actor.hasMemoryOf('exemption')
```

Comment:
could `hasMemoryOf` be part of `recalls`, so it will return either the exemption or null?